src/tabletnode/tabletnode_impl.cc

```diff
@@ -221,7 +221,8 @@ void TabletNodeImpl::LoadTablet(const LoadTabletRequest* request,
     response->set_sequence_id(request->sequence_id());
     std::string sid = GetSessionId();
     if (!request->has_session_id() ||
-        request->session_id() != sid) {
+        (sid.size() == 0) ||
+        request->session_id().compare(0, sid.size(), sid)) {
```

Comment:
because it is the style of this project

for bool:
```
    if (is_valid) {
        ...
    }
    if (!is_finished) {
        ...
    }
```

for integer:

```
    if (my_value == 0) {
        ...
    }
```

for pointer:

```
    if (p_value == NULL) {
        ...
    }
```